Add: support for the real reBot arm B601 robotic arm (#709)
NVIDIA/Isaac-GR00T: examples/rebot-arm-dm/README.md

# Finetuning GR00T N1.7 Model for Seeed reBot Arm B601 DM

This guide shows how to finetune dataset collected from [rebot arm b601 dm](https://www.seeedstudio.com/reBot-Arm-B601-DM-Bundle.html) robot, and evaluate the model on the real robot.

## Dataset

> To collect the dataset via teleoperation, please refer to the official documentation in lerobot: https://wiki.seeedstudio.com/rebot_arm_b601_dm_lerobot/#calibrate-the-robotic-arm

Reviewer: Can you add a reference to https://huggingface.co/datasets/youjiang97/organize_test_tube_0 here so users can have an example?

  --embodiment-tag NEW_EMBODIMENT \
  --model-path /tmp/rebot-arm-dm_finetune/checkpoint-10000 \
  --traj-ids 0 \
  --action-horizon 16 \
  --steps 400
```

### Evaluation Results

The evaluation produces visualizations comparing predicted actions against ground truth trajectories:

<img src="../../media/open_loop_eval_rebot_arm_dm.jpg" width="800" alt="Open-loop evaluation results showing predicted vs ground truth trajectories" />

To read these numbers and decide whether your fine-tune is working, see [Interpreting the Result: Is My Fine-tune Working?](../../getting_started/finetune_new_embodiment.md#interpreting-the-result-is-my-fine-tune-working).

## Closed-Loop Evaluation

Please refer to [eval_rebot_arm_dm.py](../../gr00t/eval/real_robot/rebot-arm-dm/eval_rebot_arm_dm.py) for how to write rebot arm deployment code using Policy API.

1. set up client side deps

```bash
cd gr00t/eval/real_robot/rebot-arm-dm
uv venv
source .venv/bin/activate
uv pip install -e . --verbose
uv pip install --no-deps -e ../../../../
```

2. Start policy server from the repository root in a separate terminal:
```bash
uv run python gr00t/eval/run_gr00t_server.py \

>   --model-path /tmp/so100_finetune/checkpoint-10000 \

Reviewer: ```suggestion
  --model-path  /tmp/rebot-arm-dm_finetune/checkpoint-10000 \
```

# Finetuning GR00T N1.7 Model for Seeed reBot Arm B601 DM

This guide shows how to finetune dataset collected from [rebot arm b601 dm](https://www.seeedstudio.com/reBot-Arm-B601-DM-Bundle.html) robot, and evaluate the model on the real robot.

## Dataset

To collect the dataset via teleoperation, please refer to the official documentation in lerobot: https://wiki.seeedstudio.com/rebot_arm_b601_dm_lerobot/#calibrate-the-robotic-arm

The commands below use the test tube organization dataset as an example

[youjiang97/organize_test_tube_0](https://huggingface.co/datasets/youjiang97/organize_test_tube_0)

## Handling the dataset

```bash
uv run --project scripts/lerobot_conversion \
  python scripts/lerobot_conversion/convert_v3_to_v2.py  \

>   --root <path-to-lerobot-dataset>

Reviewer: The conversion script requires `--repo-id` and appends it to `--root`, so `--root` is the dataset storage parent rather than the direct dataset path. The current command fails as written. Could we make both requirements explicit?

```suggestion
uv run --project scripts/lerobot_conversion \
  python scripts/lerobot_conversion/convert_v3_to_v2.py \
  --repo-id youjiang97/organize_test_tube_0 \
  --root <path-to-dataset-parent>
```

```

Then move the `modality.json` file to the root of the dataset.
```bash
cp examples/rebot-arm-dm/modality.json <path-to-lerobot-dataset>/meta/modality.json
```

## Finetuning

> Run the shared finetune launcher directly, using absolute joint positions (feel free to experiment with relative positions):

Reviewer: The prose currently contradicts `rebot_config.py`: the arm is configured with `ActionRepresentation.RELATIVE`, while only the gripper is absolute. This keeps the README aligned with the configuration used by the successful smoke test.

```suggestion
Run the shared finetune launcher directly. The provided modality config trains the six arm joints as relative actions and the gripper as an absolute action:
```
